src/tests/ftest/util/daos_core_base.py

```diff
@@ -62,13 +62,13 @@ def run_subtest(self):
                                        '/run/daos_tests/num_replicas/*')
         scm_size = self.params.get("scm_size", '/run/pool/*')
         args = self.params.get("args", '/run/daos_tests/Tests/*', "")
-        dmg = self.get_dmg_command()
-        dmg_config_file = dmg.yaml.filename
 
-        cmd = "{} {} -n {} -x D_LOG_FILE={} {} -s {} -n {} -{} {}".format(
+        cmd = "{} {} -n {} -x D_LOG_FILE={} \
+            -x D_LOG_MASK=DEBUG -x DD_MASK=mgmt,io,md,epc,rebuild \
+            {} -s {} -{} {}".format(
             self.orterun, self.client_mca, num_clients,
-            get_log_file(self.client_log), self.daos_test,
-            num_replicas, dmg_config_file, subtest, args)
+            get_log_file(self.client_log), self.daos_test, num_replicas,
+            subtest, args)
```

Comment:
(pylint-bad-continuation) Wrong hanging indentation (add 4 spaces).

Reply:
I believe this is supposed to align with format(...so something like

.format(self.orterun, self.client_mca, num_clients,
              get_log_file(...

You can run ./utils/check_modueles.sh src/tests/ftest/util/daos_core_base.py to check locally